capstone-frontend/src/components/CreateList.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import {Button, Form} from 'react-bootstrap'
+
+class CreateList extends Component {
+
+  handleSubmit = async (event) => {
+
+    event.preventDefault();
+
+    console.log("Submitted")
+
+    const newBooklist = {
+      "userID": window.localStorage.getItem("userID"),
+      "booklistName": event.target[0].value,
+    }
+
+    // Store BookList in Firebase
+    await fetch("/api/booklist", {
+      method: "POST",
+      body: JSON.stringify(newBooklist)
+    });
+
+    document.location.href="/";
```

Comment:
Would be nice to route them to the new listpage directly. To solve your issue with the sidebar not updating, you could create an onSubmit listener in App.jsx that sets the state (ie. a list of BookLists) and pass it down to your CreateList component. This would trigger an update in top level state, and your SideBar could then grab its lists from App.jsx forcing a re-render. We can discuss this further, but it's very similar to the challenge Antonio faced in his Search feature and I think the solution should be similar as well.

Reply:
It now sends you to the create ListPage, unfortunately it will have no books.